server/persistence/mysql/games.go

```diff
@@ -0,0 +1,551 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/joshprzybyszewski/cribbage/model"
+	"github.com/joshprzybyszewski/cribbage/server/persistence"
+)
+
+const (
+	// Games stores the state of a game at a given time.
+	//   Each Action will update a games state and we keep a full history of all actions.
+	// The columns act as follows:
+	// GameID is a UUID to identify a game
+	// NumActions is how many actions have occurred in the game before this one
+	// ScoreBlue, ScoreRed, and ScoreGreen are the scores for each color
+	// ScoreBlueLag, ScoreRedLag, and ScoreGreenLag are the previous scores for each color
+	// Phase is the model.Phase that the game is currently in
+	// CutCard is a number representation of the card that's been cut
+	// nolint:lll Crib is an 8-byte int of the (up to 4) cards in the crib where every two bytes is each crib card. Tricky, I know. Probably shouldn't do it this way, but yolo, this is a fun project and I can save space and mental pain
+	// CurrentDealer is the PlayerID for the dealer
+	// BlockingPlayers is a json encoded map of who's blocking and why
+	// Hands is a json encoded map of slices for player hands
+	// PeggedCards is the json-encoded slice of previously pegged cards
+	// Action is the json encoded model.PlayerAction
+	createGameTable = `CREATE TABLE IF NOT EXISTS Games (
+		GameID INT UNSIGNED,
+		NumActions INT UNSIGNED,
+		ScoreBlue TINYINT UNSIGNED,
+		ScoreRed TINYINT UNSIGNED,
+		ScoreGreen TINYINT UNSIGNED,
+		ScoreBlueLag TINYINT UNSIGNED,
+		ScoreRedLag TINYINT UNSIGNED,
+		ScoreGreenLag TINYINT UNSIGNED,
+		Phase TINYINT UNSIGNED,
+		CutCard SMALLINT,
+		Crib BIGINT,
```

Comment:
TODO: I may be able to be clever enough to store this as an `INT` instead of a `BIGINT`

Reply:
https://github.com/joshprzybyszewski/cribbage/pull/43/commits/c9f6fe178bf4dc62e427d2ec1c68cb23b86eb840